grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java

```diff
@@ -156,6 +156,7 @@ private void importDataFromFile(File file, Loader loaderParam, Future statusPrin
                 while (var.equals(MATCH_KEYWORD)) {
                     var = consumeInsertRelation(batchIterator, loaderParam);
                 }
+                loaderParam.waitToFinish();
```

Comment:
@alexandraorth can you confirm this incredibly important line won't break everything?